config/chime_science_run_recv_21cm.yaml

```diff
@@ -0,0 +1,63 @@
+##########################################
+#
+# chime_science_run_recv_21cm.yaml
+#
+# CHIME 21cm absorber receiver node configuration used in the December 2019 run.
+#
+# Author: James Willis
+#
+##########################################
+---
+type: config
+log_level: info
+num_elements: 2048
+num_local_freq: 1
+udp_packet_size: 4928
+num_data_sets: 1
+samples_per_data_set: 32768
+buffer_depth: 64
+baseband_buffer_depth: 282 # 282 = ~34 seconds after accounting for active frames
+num_gpu_frames: 128
+block_size: 32
+cpu_affinity: [1,6,7,9,14,15]
+num_frb_total_beams: 1024
+num_sub_freqs: 128
```

Comment:
It's not obvious to me that all of these values are needed.  Can we remove the ones that aren't?